install/helm/agones/templates/service/allocation.yaml

```diff
@@ -161,8 +184,19 @@ spec:
         - name: FEATURE_GATES
           value: {{ .Values.agones.featureGates | quote }}
         ports:
+        {{- if .Values.agones.allocator.service.http.enabled }}
         - name: https
-          containerPort: 8443
+          containerPort: {{ .Values.agones.allocator.service.http.targetPort }}
+        {{- if .Values.agones.allocator.service.grpc.enabled }}
+        {{- if ne .Values.agones.allocator.service.grpc.port .Values.agones.allocator.service.http.port }}
+        - name: grpc
```

Comment:
Related to the above, it might be worth having this be defined via the helm value as well.

Reply:
Done.